src/site/content/en/progressive-web-apps/add-manifest/index.md

```diff
@@ -25,11 +25,10 @@ app is launched.
 Manifest files are [supported](https://developer.mozilla.org/en-US/docs/Web/Manifest#Browser_compatibility) in Chrome, Edge, Firefox, UC Browser, Opera,
 and the Samsung browser. Safari has partial support.
 
-## Create the manifest.webmanifest file {: #create }
+## Create the manifest file {: #create }
 
-The manifest file can have any name, but is commonly named
-`manifest.webmanifest` and served from the root (your website's top-level
-directory).
+The manifest file can have any name, but is commonly named `manifest.json` and
+served from the root (your website's top-level directory).
```

Comment:
```suggestion
served from the root (your website's top-level directory). The specification
suggests the extension should be `.webmanifest`, but browsers also support
`.json` extensions, which is may be easier for developers to understand.
```

Reply:
How about adding something like the following to the doc, so that it's clear that we know about `.webmanifest`, but have explicitly chosen to use `.json`

Reply:
Makes sense to me.